src/foam/nanos/cron/DateIntervalSchedule.js

```diff
@@ -0,0 +1,101 @@
+/**
+ * @license
+ * Copyright 2021 The FOAM Authors. All Rights Reserved.
+ * http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+ foam.CLASS({
+  package: 'foam.nanos.cron',
+  name: 'DateIntervalSchedule',
+  implements: [
+    'foam.nanos.cron.Schedule'
+  ],
+
+  javaImports: [
+    'foam.core.X',
+    'java.time.LocalDateTime',
+    'java.time.ZoneId',
+    'java.util.Date',
+  ],
+
+  documentation: `
+    Schedule periodically with a minutely,hourly,daily,weekly,monthly,yearly intervals.
+  `,
+
+  properties: [
+    {
+      class: 'DateTime',
+      name: 'startDate',
+      label: 'Start Date',
+      documentation: 'Date of the first scheduled run.',
+      order: 1,
+      gridColumns: 3
+    },
+    {
+      class: 'Int',
+      name: 'intervalModifier',
+      documentation: `Modifies the interval,
+        example: for ever other week set this to 2 and interval to weekly
+      `,
+      validationPredicates: [
+        {
+          args: ['intervalModifier'],
+          predicateFactory: function(e) {
+            return e.GTE(foam.nanos.cron.DateIntervalSchedule.INTERVAL_MODIFIER, 1);
+          },
+          errorString: 'Please enter a number greater than 1.'
+        }
+      ],
+      value: 1,
+      order: 2,
+      gridColumns: 2
+    },
+    {
+      class: 'String',
+      name: 'interval',
+      documentation: 'Time between last run and new scheduled date.',
+      view: {
+        class: 'foam.u2.view.ChoiceView',
+        placeholder: 'Please select',
+        choices: ['Minutely', 'Hourly', 'Daily', 'Weekly', 'Monthly', 'Yearly']
```

Comment:
So no comments on my comments on the fact that this list incomplete and is not really very useful for payments.  
Payments are always based on a month schedule.  1st and 3rd Friday, last Friday, ...  would be much more useful than the above.  
Also, if we are providing scheduling options then we also need to account for if this is the desired payment date, so that we can schedule it the day before so it arrives on the scheduled date.